doc/version-history.rst

```diff
@@ -8,7 +8,24 @@ Version History
 .. No new work should be required in order to complete this section.
 .. Below is an example of a version history format.
 
-    In Jenkinsfile.cycle rename rubintv-broadcaster-pub -> rubintv-broadcaster-usdf
+
+Cycle 28 revision 7
+===================
+
+* Update version of the following packages:
+
+  * ts_standardscripts.
+  * ts_externalscripts.
```

Comment:
You end this and the previous line with a `.` but the next 6 not. It looks like we always end those lines in a `.` so please add them here too.